src/main/resources/com/cloudbees/hudson/plugins/folder/computed/FolderComputation/index.jelly

```diff
@@ -34,7 +33,7 @@ xmlns:fo="/lib/folder">
                     (<i:formatDate value="${it.timestamp.time}" type="both" dateStyle="medium" timeStyle="medium"/>)
                 </j:if>
             </t:buildCaption>
-            <fo:blockWrapper>
+            <table style="margin-top: 1em; margin-left:1em;">
```

Comment:
Actually it was wrong when the blockwrapper was introduced to use it here. The `t:summary` is explicitly requiring a surrounding table as it itself creates a `<tr>`
Though I couldn't find a way how to make this page show up anywhere. I think it would require having a ComputedFolder that keeps kind of an execution history of the computations but branch-api doesn't do that